packages/core/tests/acceptance/table-test.js

```diff
@@ -37,7 +45,7 @@ test('visiting /table', function(assert) {
 
   andThen(() => {
     assert.deepEqual(
-      find('.table-header-cell__title')
+      find('div.table-header-row-vc .table-header-cell__title')
```

Comment:
Do we need to be this specific?

Reply:
Yes, you have the same class inside `tr.table-header-row-vc th`, we want to verify the visible one